source/codegen/metadata/nixnetsocket/custom_proto.mako

```diff
@@ -16,3 +16,27 @@ message SockAddr {
     SockAddrIPv6 ipv6 = 2;
   }
 }
+
+enum OptName {
```

Comment:
I planned to have this enum generated and in enums.py but it wasn't picked up as a function enum during proto generation since it wasn't marked on a parameter anywhere in functions.py. I planned to wait until I add GetSockOpt which will include the enum on a param to move it over to enums.py.

Reply:
In new update it's in enums.py and generating.